mssql_base.py

```diff
@@ -5,13 +5,23 @@
 from nio.signal.base import Signal
 from nio.util.discovery import not_discoverable
 
+driver_name = ''
+
+try:
+  driver_names = [x for x in pyodbc.drivers() if x.endswith(' for SQL Server')]
+  if driver_names:
+      driver_name = driver_names[0]
+  if not driver_name:
+      raise ReferenceError('No suitable ODBC driver found. Block will not be able to connect to MSSQL DB. View README.md to install driver.')
```

Comment:
Why do we raise just to print that message? Can we just print it here? I think this can also be cleaned up a bit, maybe this:
```python
try:
  driver_name = next((x for x in pyodbc.drivers() if x.endswith(' for SQL Server')), None)
  assert driver_name
except Exception:
  print('No driver found...')
```